docs/_packages/oauth.md

```diff
@@ -288,6 +288,26 @@ const installer = new InstallProvider({
 ```
 ---
 
+### State validation
```

Comment:
I noticed that there's some inconsistency between other SDK which use the word(s) `validate, validation` and Bolt JS, where we use `verify, verifyStateParam()` etc. If others feel like being internally consistent with existing `verifyStateParam` method is more important than being consistent with Bolt Python etc, I can make modifications to change everything to `verify`

Reply:
Good point. As the `verifyStateParam` method is public, I'm fine to use `verify` over `validate` only in bolt-js. To avoid having breaking changes, I prefer using validate for other SDKs.